event-management-frontend/src/types/SublocationType.tsx

```diff
@@ -0,0 +1,5 @@
+export interface SublocationType {
```

Comment:
General comment about model / dto types: you don't need to have separate files for all of them. You can have two files (or more): model.ts, dto.ts or something like that.